packages/datadog-plugin-jest/src/index.js

```diff
@@ -36,8 +38,29 @@ function createWrapTeardown (tracer) {
   }
 }
 
+let params = {}
+
 function createHandleTestEvent (tracer, testEnvironmentMetadata) {
   return async function handleTestEventWithTrace (event) {
+    if (event.name === 'start_describe_definition') {
+      params = {}
+      shimmer.wrap(this.global.test, 'each', function (original) {
```

Comment:
Why use `shimmer` here instead of the `wrap` function from the instrumenter?

Reply:
sure, that's better 👍

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/1345/commits/36882794d13d37e55619b5a38113532976cbf66f